jenkins-integration/beaker/install/shared/81_configure_jruby_jar.rb

```diff
@@ -0,0 +1,29 @@
+require 'puppet/gatling/config'
+
+test_name 'Configure jruby jar path'
+
+def service_config_name(service_name)
+  "/etc/sysconfig/#{service_name}"
+end
+
+def set_bash_variable(host, filepath, variable_name, value)
```

Comment:
Maybe it would be better to name this function something like `set_service_environment_variable` - to indicate that the value you are setting is an environment variable that will be used by the different service commands?